packages/terra-site/src/TestContainer.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './site.scss';
+
+const TestContainer = ({ children }) => (
+  <div className={styles.app} style={{ overflow: 'auto' }}>
+    <div id="wdio-test-wrapper">
```

Comment:
This will be our base component. Not though that we still have to have individual for some tests, as they want to display in an individual component inside the wrapper.